README.md

```diff
@@ -15,6 +15,7 @@ Agent Learning Framework (ALF) is a reinforcement learning framework emphasizing
 |[PPO](alf/algorithms/ppo_algorithm.py)|On-policy RL|Schulman et al. "Proximal Policy Optimization Algorithms" [arXiv:1707.06347](https://arxiv.org/abs/1707.06347)|
 |[DDPG](alf/algorithms/ddpg_algorithm.py)|Off-policy RL|Lillicrap et al. "Continuous control with deep reinforcement learning" [arXiv:1509.02971](https://arxiv.org/abs/1509.02971)|
 |[SAC](alf/algorithms/sac_algorithm.py)|Off-policy RL|Haarnoja et al. "Soft Actor-Critic Algorithms and Applications" [arXiv:1812.05905](https://arxiv.org/abs/1812.05905)|
+|[HER](alf/algorithms/data_transformer.py)|Off-policy RL|Andrychowicz et al. "Hindsight Experience Replay" [arXiv:1707.01495](https://arxiv.org/abs/1707.01495). (see HindsightExperienceTransformer)|
```

Comment:
"(see HindsightExperienceTransformer)" Should put in the first column? The last column is the paper reference

Reply:
@hnyu I changed the link to a permalink.  Could you approve if you are Ok with this?

Thanks,
Le